libs/components/lookup/src/lib/modules/selection-modal/selection-modal.module.ts

```diff
@@ -2,6 +2,7 @@ import { NgModule } from '@angular/core';
 
 /**
  * @deprecated The `SkySelectionModalModule` is no longer needed and can be removed from your application.
+ * @internal
```

Comment:
This module is empty and deprecated, and we don't want it to appear on the docs.